src/StartupScripts/StartupManagerFactory.ts

```diff
@@ -0,0 +1,16 @@
+import { StartupManager } from './StartupManager';
+import { OsServiceScriptManager } from './OsServiceScriptManager';
+import { PlatformUnsupportedError } from './PlatformUnsupportedError';
+import { platform } from 'os';
+
+export class StartupManagerFactory {
+  public create(): StartupManager {
+    const os = platform();
+    switch (os) {
+      case 'win32' || 'linux':
```

Comment:
it's called fall-through.

```suggestion
      case 'win32':
      case 'linux':
```